components/locale-chooser.js

```diff
@@ -4,8 +4,8 @@ import {
   fallbackLocale,
   supportedLocales,
   getLocaleAndHost,
-  withContent,
```

Comment:
`withContent` will no longer be needed at all. Just `import content from './content/whatever'` :)